css/editorialBio.css

```diff
@@ -0,0 +1,12 @@
+.orcidContent{
+    display:flex;
+}
+
+.orcidIcon{
```

Comment:
These names are generic enough to be potentially used by someone else.  You can easily disambiguate our use case by specifying that they need to live within an `div.editorialTeamBioContent`.  See https://www.w3schools.com/css/default.asp for good CSS guide (e.g. our discussion on `.classes` vs. `#identifier`.